components/rating/index.css

```diff
@@ -73,9 +80,13 @@ governing permissions and limitations under the License.
   display: inline-flex;
   flex: 0 0 auto;
 
+  gap: var(--mod-rating-icon-gap, var(--spectrum-rating-icon-gap));
+
   inline-size: calc((var(--mod-rating-icon-width, var(--spectrum-rating-icon-width)) + var(--mod-rating-icon-spacing, var(--spectrum-rating-icon-spacing))) * var(--mod-rating-icon-count, var(--spectrum-rating-icon-count)));
   border-radius: var(--mod-rating-border-radius, var(--spectrum-rating-border-radius));
 
+  padding-inline: var(--mod-rating-icon-gap, var(--spectrum-rating-icon-gap));
```

Comment:
The design spec appears to have the left and right focus indicator gap as the same as the gap between the icons, so I reused the token here as `padding`